media-gfx/mypaint/mypaint-1.2.1.ebuild

```diff
@@ -46,6 +46,8 @@ src_prepare() {
 	# multilib support
```

Comment:
I do not know why the path was replaced in Scons toolchain files at first. My patch is just to fix the binary importing libs from the wrong directory after the initial partial patching.
But indeed, you can drop my PR and simply remove the multilib hack if you prefer. This would moreover follow upstream.

Reply:
Well, since you've already shown interest in fixing this package, I would prefer if you updated your PR, so I could merge it preserving the credit.